content/en/blog/2022/instrument-nginx/index.md

```diff
@@ -76,8 +76,8 @@ With the `Dockerfile` and NGINX config in place, build your docker image and run
 the container:
 
 ```console
-$ docker build -t nginx-otel --platform linux/amd64 .
-$ docker run --platform linux/amd64 --rm -p 8080:80 nginx-otel
+docker build . -t nginx-otel --platform linux/amd64 .
```

Comment:
So now this command has two dots `.`: the one just added, and one at the end.  Is that how it should be?

Reply:
Oh this is my bad, missed the suffix '.' when copying (copied line-by-line because of the $ sign). I think changing this to sh could minimize user errors but maybe this is only me